### ISSUE TITLE
If multiple photos come in too quickly printing the last ones will fail

### ISSUE THREAD

**Issue by newrooky** [newrooky/cheapstaprint#1]
If bluetooth is being used to send files to the printer then only one file can be send at a time since there is not a queuing option available on the latest versions of OSX.  This wouldn't be an issue if using windows or linux since they have available queuing options. 

Fix is sending file via USB or staggering bluetooth sending to the printer until it is completed. 

We could also mitigate the problem by sending a folder full of images to be printed on a less frequent interval, but the issue would still arrise on occasion and be too hackey.

**Comment by michaeltse** [comment]
yes, i am using apple script for the folder action to detect and send to printer via bluetooth. This happen does exist. Any fix can be done? We don not want to use cable...
